pandas/core/generic.py

```diff
@@ -7867,8 +7867,8 @@ def _doc_parms(cls):
 
     .. versionadded :: 0.22.0
 
-       Added with the default being 1. This means the sum or product
-       of an all-NA or empty series is ``NaN``.
+       Added with the default being 0. This means the sum or product
```

Comment:
can you reverse the logic of the final part here. Also can you check the consistency with other text for min_count (in this file)

Reply:
I reverse the logic in new commit. If I understand wrong, please give me specific comments. A total of 9 functions share parameter descriptions (Series.sum, Series.prod, Series.product ,  DataFrame.sum, DataFrame.prod, DataFrame.product ,  Panel.sum, Panel.prod, Panel.product).

Reply:
Do you find it clearer to phrase the second sentence as

    This means the sum of an all-NA or empty Series is 0, and the product of an all-NA or empty Series is 1.

Reply:
😄  Already made the change